src/components/Property/PropertyFlags/index.test.js

```diff
@@ -66,7 +66,7 @@ describe('PropertyFlags', () => {
       ])
     })
 
-    expect(axios).toHaveBeenCalledTimes(2)
+    // expect(axios).toHaveBeenCalledTimes(2)
```

Comment:
This should still be the case, I think.
It should make a request for the location alerts and the person alerts :thinking: 
Unless some re-renders trigger dupe requests? Meaning the count is higher.